docs/HowToFAQ/FAQCurry.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Is there a way to do partial application for functions?
+---
+
+## Question
+
+Is there a way to do partial application for functions in Dafny?
+
+## Answer
+
+Given
+```
+function f(i: int, j: int): int {...}
+```
+one can create a new partially-evaluated function, say evaluating the first argument at `2`, by writing the lambda expression `k -> f(2,k)`.
```

Comment:
I think these expressions should use `=>` instead of `->`.

Reply:
Correct.